src/Kernel32.Desktop/Kernel32.cs

```diff
@@ -4,9 +4,8 @@
 namespace PInvoke
 {
     using System;
-    using System.IO;
     using System.Runtime.InteropServices;
-    using FILETIME = System.Runtime.InteropServices.ComTypes.FILETIME;
```

Comment:
Please don't remove this `FILETIME` using. Without it, C# binds against a different `FILETIME` and we get warnings (IIRC) about using an obsolete type.